Functions/xASL_io_ReadDataPar.m

```diff
@@ -155,30 +155,38 @@
 % Iterate over fields
```

Comment:
Same question here as in other function (sorry reviewed latest commit first, going back in time :)
These things may be useful for JSONs in general, not only for DataPar?

Reply:
As previously - we shouldn't fix JSON errors in general. People should stick to JSON format. We only fix DataPar where we gave the wrong template.